devfiles/nodejs/devfile.yaml

```diff
@@ -12,7 +12,7 @@ starterProjects:
 components:
   - name: runtime
     container:
-      image: registry.access.redhat.com/ubi8/nodejs-12:1-45
+      image: registry.access.redhat.com/ubi8/nodejs-14:latest
```

Comment:
@elsony, are there any rules around container versions for stacks - is it advised that we opt for a fixed version or is it okay to pull from a `latest` tag?

Reply:
It's entirely up to the stack. When you are deciding whether you want to pull in a static version vs `latest`, you may want to consider the update strategy of your stack. If you are pulling in the latest, the benefit is any non-breaking changes that you introduce to the image will be automatically picked up by the user who has created the stack before. On the other hand, in case you are introducing breaking changes to the image (e.g. requires reacting changes to the commands in the devfile to work with it), it may potentially break existing user applications. The existing applications that were created before will refer to the latest image but they don't have the required changes on the commands on their existing devfile.

Another approach that you can consider is to introduce something like a 1.x stable image where you can still update for bug fixes/compatible changes. It will still leave room to allow you to create a 2.x stable image in case you need to introduce breaking changes to the image in the future without breaking existing applications.

Reply:
I would be a +1 for using latest in this case.  node 14 is LTS, so there shouldn't be any breaking changes at this point.  Also using latest makes it a little easier to deal with any CVE updates.  

For future node releases like Node 16,  we wouldn't send a PR to update the devfile until it is LTS anyway, so we wouldn't worry about breaking changes.

thoughts?

Reply:
Agreed. I think `latest` is okay here as we're very unlikely to pull in any changes that will require reacting changes in the DevFile. `latest` should also make maintenance a little easier for us.